postgres/tasks.py

```diff
@@ -44,8 +47,9 @@ def import_snapshot(ctx, snapshot_name=None, migrate=True):
     # Make minor adjustments to dumps for legacy dumps to work on AWS
     if migrate:
         print("Migrating dump file")
-        ctx.run(f"sed -i 's/OWNER TO surf/OWNER TO postgres/g' {snapshot_file_path}", echo=True)
+        ctx.run(f"sed -i.bak 's/OWNER TO surf/OWNER TO postgres/g' {snapshot_file_path}", echo=True)
 
     print("Importing snapshot")
     ctx.run(f"psql -h localhost -U postgres -d edushare -f {snapshot_file_path}",
             pty=True, watchers=[postgres_password_responder])
+    ctx.run(f"rm {snapshot_file_path}.bak", warn=True)
```

Comment:
Voor Linux versie is deze nog wel handig, anders krijg ik heel veel zwervende grote data bestanden, die ik nooit gebruik.

Reply:
Oh, die file blijft inderdaad staan. Dan is dit zeker een goede toevoeging.